### ISSUE TITLE
Media is not shown in favourite and boost notifications

### ISSUE THREAD

**Issue by MightyPork** [tootsuite/mastodon#5113]
Example:

![screenshot_20170926_141857](https://user-images.githubusercontent.com/2041118/30859894-ab705d30-a2c5-11e7-9ea0-fca9671a79c6.png)

If this "works as intended", it's confusing, especially when you didn't put anything very descriptive in the text.

**Comment by Gargron** [comment]
Yeah it's intended, it's the "muted" prop on the Status component.

**Comment by MightyPork** [comment]
i'm thinking there could be some sort of spoiler/collapsed view of the media, so you don't have to click for Detailed View when you're curious what it was

**Comment by nightpool** [comment]
Isn't that basically what detailed view IS? The full height for the
collapsed view?

**Comment by Cassolotl** [comment]
Notifications for toots with media have changed a little, since media URLs are no longer automatically included in toots. They currently look like this:

![screen shot 2018-05-26 at 13 09 59](https://user-images.githubusercontent.com/6607512/40575931-25886b88-60e6-11e8-8a07-d1f9e85ecb83.png)
